mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,17 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return `Hello, my {name}` is "and I am $age years old`;
+function introduceMe(name, age) {
+  return `Hello, my name is ${name} and I am ${age} years old`;
+}
 
```

Comment:
Small comment on this one - whenever you are declaring a new variable, it's a good idea to use `let` or `const`.